tests/unit/helpers.js

```diff
@@ -442,9 +442,9 @@ export function createFakeUserAbuseReport({
 // if (fakeConfig.get('isDevelopment')) {
 //   ...
 // }
-export const getFakeConfig = (params = {}) => {
+export const getFakeConfig = (params = {}, forceKey = false) => {
```

Comment:
Because you never only add one option, I'd rather see it as an object. This also makes calling code easier to read.

```js
export const getFakeConfig = (params = {}, { forceKey = false } = {}) => {
  // ...
}
```

Additionally, since this function will be called in test code which doesn't have Flow coverage, optional positional arguments are dangerous (they are hard for refactoring).

Reply:
works for me